README.rst

```diff
@@ -83,7 +83,7 @@ If you run this command the first time, a new database and the admin
 account (Username: `admin`, Password: `admin`) will be created. Please
 change the password after first login!
 
-OpenSlides will start using the integrated Tornado webserver. It will also
+OpenSlides will start a webserver. It will also
```

Comment:
Proposal: Fit all lines of this paragraph to the default line ending (`Ctrl+J` in Geany).